monster-cards/src/widgets/card/createCardSummary.ts

```diff
@@ -0,0 +1,64 @@
+import createRenderMixin, { RenderMixin, RenderMixinOptions, RenderMixinState } from 'dojo-widgets/mixins/createRenderMixin';
+import createRenderableChildrenMixin from 'dojo-widgets/mixins/createRenderableChildrenMixin';
+import createStatefulChildrenMixin, { StatefulChildrenState, StatefulChildren } from 'dojo-widgets/mixins/createStatefulChildrenMixin';
+import createCard from './createCard';
+import createWidget from 'dojo-widgets/createWidget';
+import { Child } from 'dojo-widgets/mixins/interfaces';
+
+export type CardSummaryState = RenderMixinState & StatefulChildrenState & {
+	name: string;
+	cardImage: string;
+	score: number;
+	id: string;
+	cardId: string;
+}
+
+type CardSummaryOptions = RenderMixinOptions<CardSummaryState>;
+
+export type CardSummary = RenderMixin<CardSummaryState> & StatefulChildren<Child>;
+
+const create = createRenderMixin
```

Comment:
I'd been naming them all `create`. do you feel there is a need to rename the `create` function inside `createCardSummary` to `createCardSummary` ?